tap_github/__init__.py

```
@@ -1098,6 +1135,33 @@ def do_sync(config, state, catalog):
 
                 singer.write_state(state)
 
+
+def get_repos_from_config(config):
+    organization = config.get('organization')
+    repos_include = list(
+        filter(None, ' '.join(filter(None, (config.get('repository'), config.get('repos_include')))).split(' '))
+    )
+    repos_exclude = list(filter(None, config.get('repos_exclude', '').split(' ')))
+
```

what happens if both `organization` and `repository` are empty or not exist in the config?

can we raise an exception with a human readable error message to cover that case?

---

+1, this function doesn't have any protection against `organization` not present.

---

Added a validation for args